src/System.Net.HttpListener/tests/HttpListenerContextTests.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    public class HttpListenerContextTests : IDisposable
+    {
+        private HttpListenerFactory Factory { get; }
+        private ClientWebSocket Socket { get; }
```

Comment:
Where is this used?  It's a good idea to have this and to have tests that use it to connect, and validate that we can actually connect and have a web socket conversation between the client and server, but I don't see such tests here.